apps/charterafrica/src/components/Mooc/Mooc.js

```diff
@@ -0,0 +1,100 @@
+import { Section } from "@commons-ui/core";
+import { Figure } from "@commons-ui/next";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import React from "react";
+
+import { neutral } from "@/charterafrica/colors";
+import LineClampedRichTypography from "@/charterafrica/components/LineClampedRichTypography";
+
+const Mooc = React.forwardRef(function Mooc(props, ref) {
+  const { title, link, image, sx } = props;
+
+  if (!title || !link || !image) {
+    return null;
+  }
+
+  return (
+    <Box bgcolor={neutral[900]} ref={ref} sx={sx}>
+      <Section
+        ref={ref}
+        sx={{ px: { xs: 5, sm: 0 }, py: { xs: 5, md: "86px" } }}
+      >
+        <Box
+          display="flex"
+          flexDirection={{ xs: "column-reverse", sm: "row" }}
+          gap={5}
+          justifyContent="center"
+          alignItems="center"
+        >
+          <Box
+            flex={1.2}
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            alignItems={{ xs: "center", md: "flex-start" }}
+            sx={{ gap: 5 }}
+          >
+            <LineClampedRichTypography
```

Comment:
If we're using `LineClampedRichTypography`, I expect to see `lineClamp` prop. If not, why are we using this over a normal `RichTypography` or `RichText`?